src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -82,7 +81,7 @@ interface IState {
     downloaded: boolean;
     setPassphrase: boolean;
     backupInfo: IKeyBackupInfo | null;
-    backupSigStatus: TrustInfo | null;
+    backupEnabled: boolean | null;
```

Comment:
can we (throughout this PR) document what, exactly, "enabled" means? Does it mean we are actively backing up keys to the server? Will we automatically request keys from the server? Or do we just have the potential to do those things?

(It seems like it means different things in different places? If we can't find better names then we'll need to make sure we document it clearly in each location.)

Reply:
I'll try to clarify but the API is confusing currently. Deprecated API was using the term `usable` now we use only `trusted`.
And enabled means that it's capable of backing up your keys (not restoring).
It's a bit more complex due to the extensible nature of backups via different algorithms.
But a trusted assymetric backup is usable and thus used. A symmetric backup could be trusted but not usable because we don't have the key to encrypt.